system-addon/content-src/components/TopSites/_TopSites.scss

```diff
@@ -341,19 +377,26 @@
     }
 
     .section-title {
-      margin-bottom: 5px;
+      text-transform: none;
```

Comment:
Probably good to double check if this was an intended design change to avoid the upper casing that is used for all other section titles requiring overriding here.

Reply:
Yes, @aaronrbenson 
> the all-caps version just doesn’t look good in this case
> trying to use all-caps very sparingly